.github/workflows/github-ci.yml

```diff
@@ -406,3 +409,34 @@ jobs:
           mini-apps/ionosphereSolverTests/LFMtest.png
           mini-apps/ionosphereSolverTests/atmosphere.png
         if-no-files-found: error
+
+  check_cfg_files:
+    runs-on: ubuntu-latest
+    container: ursg/vlasiator_ci:20240131_1
+    needs: build_production
+    steps:
+    - name: Checkout source
+      uses: actions/checkout@v4
+      with:
+        submodules: false
+    - name: Download release binary
+      uses: actions/download-artifact@v4
+      with:
+        name: vlasiator-release
+    - name: Make release binary executable
+      run: |
+        chmod +x $GITHUB_WORKSPACE/vlasiator
+    - name: Check cfg files in projects/
+      run: |
+        retval=0
+        for cfg in `find projects -name \*.cfg`; do
```

Comment:
Include samples and testpackage while at it?

Reply:
Done